org.duckstation.DuckStation.yml

```diff
@@ -0,0 +1,54 @@
+app-id: org.duckstation.DuckStation
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: duckstation-nogui
+finish-args:
+  - --device=all
+  - --filesystem=host:ro
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+
+modules:
+  - name: libevdev
+    buildsystem: meson
+    config-opts:
+      - -Dtests=disabled
+      - -Ddocumentation=disabled
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share
+    sources:
+      - type: archive
+        url: https://www.freedesktop.org/software/libevdev/libevdev-1.11.0.tar.xz
+        sha256: 63f4ea1489858a109080e0b40bd43e4e0903a1e12ea888d581db8c495747c2d0
+
+  - name: duckstation
+    no-make-install: true
```

Comment:
you forgot `buildsystem: cmake-ninja` here. This is what cause the build failure.